tabuckner/broskichat
# Integrated header

**Project Changed**
-App

**- What I did**
-Implemented the header component
-Implemented really simple testing that checks the value of a variable called logoTitle

**- How I did it**
-Added path /src/Components/Partials/*
-Added package 'react-test-renderer'
-Added simple test to make sure a variable is equal to "Broski Test" inside /src/App.test.js

**- How to test verify it**
-Clone the "integrated-header" branch. You will see the updated simple header.
-For testing make sure the variable "logoTitle" inside ../Partials/Header.js is equal to the string specified in the test file

**- An image to make us giggle**
-Nothing this time :(
